cice/Release-5.1/comp_ice.esmf

```diff
@@ -30,9 +30,14 @@ if ($RES == 'col') then   # for column configuration:
    setenv NSNWLYR    1       # number of vertical layers in the snow
    setenv NICECAT    5       # number of ice thickness categories
 else                      # test problem: gx3
-   setenv NTASK     30       # total number of processors
-   setenv BLCKX     50       # x-dimension of blocks ( not including )
-   setenv BLCKY     50       # y-dimension of blocks (  ghost cells  )
+#   setenv NTASK     30       # total number of processors
+#   setenv BLCKX     50       # x-dimension of blocks ( not including )
+#   setenv BLCKY     50       # y-dimension of blocks (  ghost cells  )
+#
```

Comment:
Unless there is s a specific reason for keeping them, I suggest we remove the commented out lines.